build/configs/qemu/README.md

```diff
@@ -4,7 +4,9 @@ QEMU is a generic and open source machine emulator and virtualizer.
 
 ## Information
```

Comment:
Delete this information bloc here. First should be Environment Set-Up with QEMU installation.